src/types.rs

```diff
@@ -15,7 +15,7 @@ impl Hue {
 
     /// Return a hue value in the interval [0, 360].
     pub fn value(self) -> Scalar {
-        if self.unclipped == 360.0 {
+        if (self.unclipped - 360.0).abs() < std::f64::EPSILON {
```

Comment:
Same here. Please leave the `==` check in place.

Reply:
Commit dropped (reverted).